management-service/src/main/java/pl/pwr/zpi/reports/repository/SchedulerRepository.java

```diff
@@ -0,0 +1,7 @@
+package pl.pwr.zpi.reports.repository;
+
+import org.springframework.data.jpa.repository.JpaRepository;
+import pl.pwr.zpi.reports.dto.scheduler.ClusterSchedule;
+
+public interface SchedulerRepository extends JpaRepository<ClusterSchedule, String> {
```

Comment:
I believe that good practice is to name repository as $entity_nameRepository, so you'd know what entity belongs to this repo by reading the name